src/app/modules/main/wallet_section/transactions/controller.nim

```diff
@@ -50,7 +50,7 @@ proc init*(self: Controller) =
 
   self.events.on(SIGNAL_HISTORY_READY) do (e:Args):
     let args = HistoryArgs(e)
-    self.delegate.setHistoryFetchState(args.addresses, isFetching = false)
+    self.delegate.setHistoryFetchState(args.addresses, isFetching = true)
```

Comment:
in what case do we set this to false then?

Reply:
good call, when non archival node or when tx are loaded